### DISCUSSION TITLE
Short question maybe stupid, why pg-strom named strom?

### DISCUSSION THREAD

**Discussion by matrixji** [heterodb/pg-strom#568] [General]
A newbie user of pg-strom, previously I pronounce it as pg-storm, and recently I realize I misspelling it.

**Comment by kaigai** [comment]
GPU's core is named "streaming multiprocessor" (shortly, SM).
PG-Strom originally means for PostgreSQL's extension to utilize streaming multiprocessors.
Strom is Germany term for "Stream", where I lived in when I started its development.
